setup.py

```diff
@@ -51,6 +50,7 @@
         "eth-utils>=1,<2",
         "eth-hash[pycryptodome]",
         "mypy-extensions>=0.4.1,<1.0.0",
+        "ruamel.yaml==0.15.87",
```

Comment:
Can we set a looser version? e.g., `"ruamel.yaml<=0.15"` as [recommendation for production](https://pypi.org/project/ruamel.yaml/)
```suggestion
        "ruamel.yaml<=0.15",
```